Connectors/Revit/Speckle.Connectors.RevitShared/Operations/Send/Settings/ToSpeckleSettingsManager.cs

```diff
@@ -102,6 +103,22 @@ public bool GetSendParameterNullOrEmptyStringsSetting(SenderModelCard modelCard)
     return returnValue;
   }
 
+  public bool GetLinkedModelsSetting(SenderModelCard modelCard)
+  {
+    var value = modelCard.Settings?.First(s => s.Id == "includeLinkedModels").Value as bool?;
+    var returnValue = value != null && value.NotNull();
+
+    if (_sendLinkedModelsCache.TryGetValue(modelCard.ModelCardId.NotNull(), out bool? previousValue))
+    {
+      if (previousValue != returnValue)
+      {
+        EvictCacheForModelCard(modelCard);
```

Comment:
I am not sure about it yet. Could you put here a note and somewhere we won't forget about that we will evaluate our cache invalidation for this setting later once we have more understanding

Reply:
Added a note above the method 👍🏼 I had just added it as a placeholder. 

See `docs: cache invalidation note` commit